openmetadata-ui/src/main/resources/ui/playwright/support/glossary/Glossary.ts

```diff
@@ -53,8 +75,16 @@ export class Glossary {
   }
 
   async create(apiContext: APIRequestContext) {
+    const apiData = {
+      name: this.data.name,
+      displayName: this.data.displayName,
+      description: this.data.description,
+      reviewers: this.data.reviewers,
+      tags: this.data.tags,
+      mutuallyExclusive: this.data.mutuallyExclusive,
+    };
```

Comment:
instead of creating a new object, I think we should just omit fqn and pass all the data we have, in that way in the future if we add any new value it will be included automatically